feat(curriculum): dice game project (#50819)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/649f03f6f661fb01d98fcfd1.md

---
id: 649f03f6f661fb01d98fcfd1
title: Step 13
challengeType: 0
dashedName: step-13
---

# --description--

Targetting the `dice` argument, you will need to apply the `textContent` property and set it to `displayArray[index]`, so that you can see the results of the dice roll in the HTML.

# --hints--

> You should apply the `textContent` property to the `dice` argument

Reviewer: ```suggestion
You should access the `textContent` property of the `dice` argument.
```

```js
assert.match(code, /dice\s*\.\s*textContent\s*/);
```

> You should set the `textContent` property to `displayArray[index]`

Reviewer: ```suggestion
You should set the `textContent` property to `displayArray[index]`.
```
